.travis.yml

```diff
@@ -0,0 +1,10 @@
+sudo: required # Required for apt-get build-dep
+language: cpp
+compiler: gcc
+
+before_install:
+    - sudo apt-get -qq update
+    - sudo apt-get build-dep -qq mumble
+
+script:
+    - qmake CONFIG+="g15-emulator qt4-legacy-compat" -recursive && make -j2
```

Comment:
Should we add 'release' here? I am always confused by Qt's default, since it depends on how your Qt was built.